recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "1.11.4" %}
+{% set version = "1.10.8" %}
```

Comment:
Again this is a downgrade. If you don't answer my messages I'll keep closing this and probably block this.

Reply:
@ocefpaf : I am sorry for your inconvenience. I am trying to build 1.10.8, and that's why I did that. It seems I am doing it in a wrong way and would like to know what's the right way to do that.

Reply:
Just read me messages. Also, please keep the discussion in a single PR. Opening multiples PRs and discussing on them is bad practices.